server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -96,74 +30,40 @@ export class AlfredResources implements core.IResources {
 		public config: Provider,
 		public producer: core.IProducer,
 		public redisConfig: any,
-		public clientManager: core.IClientManager,
-		public webSocketLibrary: string,
-		public orderManager: core.IOrdererManager,
 		public tenantManager: core.ITenantManager,
 		public restTenantThrottlers: Map<string, core.IThrottler>,
 		public restClusterThrottlers: Map<string, core.IThrottler>,
-		public socketConnectTenantThrottler: core.IThrottler,
-		public socketConnectClusterThrottler: core.IThrottler,
-		public socketSubmitOpThrottler: core.IThrottler,
-		public socketSubmitSignalThrottler: core.IThrottler,
 		public singleUseTokenCache: core.ICache,
 		public storage: core.IDocumentStorage,
 		public appTenants: IAlfredTenant[],
 		public mongoManager: core.MongoManager,
 		public deltaService: core.IDeltaService,
 		public port: any,
 		public documentsCollectionName: string,
-		public metricClientConfig: any,
 		public documentRepository: core.IDocumentRepository,
 		public documentDeleteService: IDocumentDeleteService,
-		public throttleAndUsageStorageManager?: core.IThrottleAndUsageStorageManager,
-		public verifyMaxMessageSize?: boolean,
-		public redisCache?: core.ICache,
-		public socketTracker?: core.IWebSocketTracker,
 		public tokenRevocationManager?: core.ITokenRevocationManager,
 		public revokedTokenChecker?: core.IRevokedTokenChecker,
-		public collaborationSessionEvents?: TypedEventEmitter<ICollaborationSessionEvents>,
 		public serviceMessageResourceManager?: core.IServiceMessageResourceManager,
 		public clusterDrainingChecker?: core.IClusterDrainingChecker,
 	) {
-		const socketIoAdapterConfig = config.get("alfred:socketIoAdapter");
 		const httpServerConfig: services.IHttpServerConfig = config.get("system:httpServer");
-		const socketIoConfig = config.get("alfred:socketIo");
 		const nodeClusterConfig: Partial<services.INodeClusterConfig> | undefined = config.get(
 			"alfred:nodeClusterConfig",
 		);
 		const useNodeCluster = config.get("alfred:useNodeCluster");
 		this.webServerFactory = useNodeCluster
-			? new services.SocketIoNodeClusterWebServerFactory(
-					redisConfig,
-					socketIoAdapterConfig,
-					httpServerConfig,
-					socketIoConfig,
-					nodeClusterConfig,
-			  )
-			: new services.SocketIoWebServerFactory(
-					this.redisConfig,
-					socketIoAdapterConfig,
-					httpServerConfig,
-					socketIoConfig,
-			  );
+			? new services.NodeClusterWebServerFactory(httpServerConfig, nodeClusterConfig)
+			: new services.BasicWebServerFactory(httpServerConfig);
 	}
 
 	public async dispose(): Promise<void> {
 		const producerClosedP = this.producer.close();
 		const mongoClosedP = this.mongoManager.close();
-		const tokenRevocationManagerP = this.tokenRevocationManager
```

Comment:
tokenRevocationManager is being started in runner, so we should still be closing, right?